src/components/LmsConfigurations/CanvasIntegrationConfigForm.jsx

```diff
@@ -0,0 +1,273 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import isEmpty from 'lodash/isEmpty';
+import { ValidationFormGroup, Input, StatefulButton, Icon, Button } from '@edx/paragon';
+import { snakeCaseFormData } from '../../utils';
+import StatusAlert from '../StatusAlert';
+import LmsApiService from '../../data/services/LmsApiService';
+import NewRelicService from '../../data/services/NewRelicService';
+
+export const REQUIRED_CONFIG_FIELDS = [
+  'clientId',
+  'clientSecret',
+  'canvasAccountId',
+  'active',
```

Comment:
oh seeing down further where there is no invalid label on the active field, I'm not sure this would ever get triggered anyways. You probably should remove active from this list.